client/src/containers/Login.js

```diff
@@ -82,43 +90,34 @@ class MultiFactorApp extends Component {
    * @return {null|Register}
    */
   renderRegister() {
-    const { schema, loginCompleted } = this.state;
+    const { schema, verificationCompleted } = this.state;
 
-    if (!schema || (!loginCompleted && schema.registeredMethods.length)) {
+    if (!schema || (!verificationCompleted && schema.registeredMethods.length)) {
       return null;
     }
 
-    return <Register {...schema} />;
+    return (
+      <Provider store={store}>
+        <Register {...schema} />
+      </Provider>
+    );
   }
 
   /**
-   * @return {null|Login}
+   * @return {null|Verify}
    */
-  renderLogin() {
-    const { schema, loginCompleted } = this.state;
+  renderVerify() {
+    const { schema, verificationCompleted } = this.state;
 
-    if (!schema || loginCompleted || !schema.registeredMethods.length) {
+    if (!schema || verificationCompleted || !schema.registeredMethods.length) {
       return null;
     }
 
     return (
-      <Login {...schema} onCompleteLogin={this.handleCompleteLogin} />
+      <Verify {...schema} onCompleteVerification={this.handleCompleteVerify} />
```

Comment:
Register has a store provided, does Verify need one too?

Reply:
No (not yet). Only `Register` controls its flow with Redux right now.